src/sentry/static/sentry/app/components/onboardingWizard/getOnboardingTasks.tsx

```diff
@@ -32,8 +34,8 @@ export default function getOnboardingTasks(organization: Organization): Onboardi
       detailedDescription: t('Choose your platform and send an event.'),
       skippable: false,
       prereq: [1],
-      featureLocation: 'project',
-      location: 'settings/install/',
+      actionType: 'app',
+      location: `/settings/${organization.slug}/projects/:projectId/install/`,
```

Comment:
What should happen when the first step (creating a project) hasn't been completed yet? Previously, the user was able to create a project from this location as well.